templates/email/nottinghamshirepolice/submit.html

```diff
@@ -0,0 +1,69 @@
+[%
+
+email_summary = "A new problem in your area has been reported by a " _ site_name _ " user.";
+email_footer = "If there is a more appropriate email address for messages about " _ ( report.get_extra_metadata('original_category') OR report.category ) _ ", please let us know. This will help improve the service for local people. We also welcome any other feedback you may have.";
```

Comment:
Do we want to include this message, we probably don't want this to happen for this - and they can work out how to contact the police if they have an issue, I assume.